CHANGELOG.md

```diff
@@ -1,5 +1,14 @@
 # Changelog
 
+## [9.5.1](https://github.com/theforeman/puppet-dns/tree/9.5.1) (2023-01-31)
+
+[Full Changelog](https://github.com/theforeman/puppet-dns/compare/9.5.0...9.5.1)
+
+**Fixed bugs:**
+
+- Remove legacy fact usage [\#225](https://github.com/theforeman/puppet-dns/pull/225) ([smortex](https://github.com/smortex))
```

Comment:
In case you're interested: Puppet 8 will disable legacy facts by default, so these things may pop up in more modules.